### ISSUE TITLE
update json_annotation dependency

### ISSUE THREAD

**Issue by EmanueleVinci** [FlutterGen/flutter_gen#93]
When I run "pub get" appears this error

```
Because flutter_gen >=2.0.1 <2.0.2 depends on flutter_gen_core ^2.0.1 and flutter_gen >=2.0.2 depends on flutter_gen_core ^2.0.2, flutter_gen >=2.0.1 requires flutter_gen_core ^2.0.1.
And because flutter_gen_core >=2.0.1-dev.0 depends on json_annotation ^3.1.0, flutter_gen >=2.0.1 requires json_annotation ^3.1.0.
So, because your_project depends on both json_annotation ^4.0.0 and flutter_gen ^2.0.1, version solving failed.
```

Can you please update the dependency?

**Comment by wasabeef** [comment]
@EmanueleVinci 
Thanks for using FlutterGen.
Which one are you using flutter_gen version? We released flutter_gen `2.0.2`.


dart
```
dart pub global activate flutter_gen
```

brew
```
brew upgrade fluttergen/tap/fluttergen
```

build_runner
```yaml
dev_dependencies:
  build_runner:
  flutter_gen_runner: ^2.0.2
```

**Comment by wasabeef** [comment]
@EmanueleVinci 
I understood the issue, but we have to release a null safety version to solve it.
So could you wait for the null safety version `3.0.0` maybe we will release it in a few days.

**Comment by EmanueleVinci** [comment]
ok, thank you!

**Comment by wasabeef** [comment]
@EmanueleVinci 
I'm sorry for late reply.

Just now we released `v3.0.2` null safety so could you try it?
